### ISSUE TITLE
Unable to Install HLS Downloader Plugin in Brave Browser Using .crx File

### ISSUE THREAD

**Issue by lunar-shadow** [puemos/hls-downloader#337]
**Describe the bug**
Unable to install the HLS Downloader plugin in Brave browser by using the .crx file provided on the GitHub repository. Brave is disabling the extension, which is consistent with its security measures.

**To Reproduce**

1. Download the HLS Downloader .crx file from the releases page: https://github.com/puemos/hls-downloader/releases
2. Open the Brave browser.
3. Navigate to brave://extensions/
4. Drag and drop the downloaded .crx file onto the extensions page.
5. Observe the message indicating that Brave has disabled the extension.

**Expected behavior**
I expected to be able to install HLS Downloader plugin using the .crx file without any issues.

**Screenshots**
NA

**Desktop (please complete the following information):**

- OS: Mac OS 14.4.1 
- Browser: Brave
- Version: 1.65.123 Chromium: 124.0.6367.91 (Official Build) (arm6]

**Smartphone (please complete the following information):**
NA

**Additional context**
According to Brave's support documentation (https://support.brave.com/hc/en-us/articles/360017914832-Why-am-I-seeing-the-message-extensions-disabled-by-Brave?hl=en-US), extensions may be disabled if they didn't come from the Chrome Web Store or if they were determined to be unsafe.

**Comment by puemos** [comment]
Ca you try to install the zip?

**Comment by mervynlobo** [comment]
> Ca you try to install the zip?

Yes, I tried the suggested method and it works perfectly. Here are the steps for others who might need them:
1. Download the zip file from the releases page.
2. Unzip the file to extract the `dist` folder.
3. Drag and drop the `dist` folder to the extensions page.

Since the issue is resolved for me, could you please close this issue? Thank you!

**Comment by NerOfTheRings** [comment]
I have the same problem both on Brave and Microsoft Edge

**Comment by trimechee** [comment]
I can add crx file to brave and edge, jut download crx file and rename crx with zip, may be Mr @puemos can add this awesome trick to FAQ

**Comment by NerOfTheRings** [comment]
> I can add crx file to brave and edge, jut download crx file and rename crx with zip, may be Mr @puemos can add this awesome trick to FAQ

Thanks dude it worked

**Comment by Jorman** [comment]
Hi all, sorry for the intrusion.
I realized that I was still using an old version, 3.3.0 and looked for the update, finding just 4.1.1
I tried to install it in chrome with the recommended method, i.e. dragging it to Dev mode, but then the extension was unusable and not activatable.
I then did as with the old version, extracted the .crx file and from chrome's extension manager loaded the unpackaged extension, and now it works

Maybe is not the best way but works, without change browser

**Comment by trimechee** [comment]
Great very instructive ! Thank you so much !
